trigger-and-filter/components/filter-instance-detail/filter-instance-detail.component.html

```diff
@@ -0,0 +1,41 @@
+<xc-panel class="panel">
+    <header>
+        <div class="actions">
+            <xc-button (click)="enable()" [disabled]="busy" xc-i18n xc-tooltip="tooltip-enable">
+                <xc-icon xc-icon-material xc-i18n xc-icon-name="play_arrow">enable</xc-icon>
+            </xc-button>
+            <xc-button (click)="disable()" [disabled]="busy" xc-i18n xc-tooltip="tooltip-disable">
+                <xc-icon xc-icon-material xc-i18n xc-icon-name="stop">disable</xc-icon>
+            </xc-button>
+            <xc-button (click)="undeploy()" [disabled]="busy" xc-i18n xc-tooltip="tooltip-undeploy">
+                <xc-icon xc-i18n [xc-icon-name]="XDSIconName.DELETE">undeploy</xc-icon>
+            </xc-button>
+        </div>
+    </header>
+    <section class="section" *ngIf="detail">
+        <dl class="trigger-side">Trigger
```

Comment:
description lists do not allow plain text in their content. Since this is not specified, it might be that some Browsers won't display this text properly (now or someday)
See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl?retiredLocale=de